### ISSUE TITLE
Support the Google Analytics 4 Measurement ID

### ISSUE THREAD

**Issue by DrDave** [chrometoasters/ga-gtm-singlefield#4]
Allow for the use of the new ‘Measurement ID’ that’s been introduced with the latest edition of Goole Analytics (GA4). 

Refer https://support.google.com/analytics/answer/7372977?hl=en#:~:text=A%20Measurement%20ID%20uses%20the,your%20Google%20Analytics%204%20property.

Measurement ID uses the format `G-XXXXXXX` whereas GA3 was `UA-000000-1`

The snippet that surrounds the Measurement ID (when integrated) might need to be updated, too. Please investigate and confirm whether we use `gtag.js` already or not.

**Comment by Incisiveranking2023** [comment]
Yes, I can help you with the GA4 setup. I will need your website url first so as to check the current situation, and then I will setup the required events on your website.

Let's **_[get [connected]_**](https://incisiveranking.com/contact/) today and let me sort out your issue
